docs/source/development/traffic_ops_api/v12/cachegroup_fallbacks.rst

```diff
@@ -0,0 +1,209 @@
+  +===================================+========+==========================================================================+
+  |                                   | array  | parameters array                                                         |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>cacheGroupId``                 | int    | Cache group id                                                           |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>fallbackId``                   | int    | fallback cache group id                                                  |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>cacheGroupName``               | string | Cache group name                                                         |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>fallbackName``                 | string | Fallback cache group  name                                               |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>fallbackOrder``                | int    | Ordering list in the list of backups                                     |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``alerts``                        | array  | A collection of alert messages.                                          |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>level``                        | string | Success, info, warning or error.                                         |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>text``                         | string | Alert message.                                                           |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+
+
+  **Response Example** ::
+
+    {
+       "alerts": [
+          {
+             "level":"success",
+             "text":"Backup configuration for 1 successful."
+          }
+       ],
+       "response": [
+          {
+             "cacheGroupId":1,
+             "cacheGroupName":"GROUP1",
+             "fallbackId":3,
+             "fallbackName":"GROUP2",
+             "fallbackorder":10,
+          }
+       ]
+    }
+   
+|
+
+**DELETE /api/1.2/cachegroup_fallbacks**
+
+  Delete fallback list assigned to the cache group.
```

Comment:
You can also delete one, right? Can you update the docs to reflect that? For example:

DELETE /api/1.2/cachegroup_fallbacks?cacheGroupId=3&fallbackId=23